common/types/message/message.go

```diff
@@ -71,6 +66,7 @@ type ChunkInfo struct {
 	DataHash      common.Hash `json:"data_hash"`
 	IsPadding     bool        `json:"is_padding"`
 	TxBytes       []byte      `json:"tx_bytes"`
+	TxBytesHash   common.Hash `json:"tx_data_digest"`
```

Comment:
It seems that this new field is not handled anywhere within coordinator, thus, could it be removed for conciseness?

Reply:
This field is accessed within prover circuit